src/ASM/FiniteElement.h

```diff
@@ -11,8 +11,7 @@
 //!
 //==============================================================================
 
-#ifndef _FINITE_ELEMENT_H
-#define _FINITE_ELEMENT_H
+#pragma once
```

Comment:
Er det noen god grunn til å skifte til pragma once her istedet for ifndef-define som er brukt ellers (i kjerne-koden ihvertfall)?